tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -362,7 +362,7 @@ public function testPluckReturnsTheDateAttributesOfAModel()
         $this->assertEquals(['date_2010-01-01 00:00:00', 'date_2011-01-01 00:00:00'], $builder->pluck('created_at')->all());
     }
 
-    public function testPluckWithoutModelGetterJustReturnTheAttributesFoundInDatabase()
+    public function testPluckWithoutModelGetterJustReturnsTheAttributesFoundInDatabase()
```

Comment:
This seems completely unrelated to this PR

Reply:
It is. Closed this one and split it up into separate PRs.